packages/raml-generator/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@commercetools-docs/raml-generator",
```

Comment:
The name "RAML generator" is a bit confusing for this package. I would suggest to find a more appropriate name, like `ramldoc-generator`.